spec/features/data_access/access_option_defaults_spec.rb

```diff
@@ -39,16 +39,15 @@
     before :each do
       user = User.find_or_create_by(echo_id: echo_id)
       options = {
-        accessMethod: [{method: 'FtpPushPull', model: '<broken/>', rawModel: '<broken/>', type: 'order'}]
+        'accessMethod' => [{'method' => 'stale', 'model' => '<broken/>', 'rawModel' => '<broken/>', 'type' => 'order'}]
       }
-      AccessConfiguration.set_default_options(user, collection_id, options)
+      AccessConfiguration.set_default_access_config(user, collection_id, options, ['form data changed'])
       load_page 'data/configure', project: [collection_id]
     end
 
     it "presents default options for the order" do
       expect(page).to have_unchecked_field('Download')
-      expect(page).to have_checked_field('FtpPushPull')
```

Comment:
I never noticed this line before, but it should have never been in the baseline. Ugh.

Also, I think there needs to be another test here. This spec tests that an order form that doesn't exist doesn't cause problems. We need another one where an order option that exists but has changed doesn't cause problems.